.github/ISSUE_TEMPLATE/test-failure.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Issue Template for Reporting Testing Failures
+about: Describe the failure you encountered while testing
+title: "[Test Case ID]: descriptive title"
+labels: ''
+assignees: ''
+---
+
+**Expected behavior**
+Describe what was expected to happen during testing.
+
+**Actual behavior**
+Describe what actually happened during testing, including any errors or failures encountered.
+
+**Steps to reproduce**
+Provide detailed steps to reproduce the testing failure.
+
+**Release**
+Include information about the release in which the testing was performed.
+
+**Additional information**
+Include any additional information that may be helpful in understanding the testing failure, such as screenshots or log files.
+
+**Possible solution**
+If you have any ideas or suggestions for resolving the testing failure, please include them here.
+
+**Assignee**
+If you have a particular developer or team member in mind who should investigate this issue, specify them here.
+
+**Labels**
+Specify any labels that may be helpful in categorizing or prioritizing this issue, such as "testing", "bug", or "critical".
```

Comment:
I think the **Labeles** section might be redundant, we typically lable all testings issues, as `<release.name testing>`.

Reply:
I think it is useful to ask a reporter to do this categorization.

Reply:
Yeah add more info can be useful, but calling the section *Labeles* might not be the best, _assuming we have strict policiy for labels_

Reply:
You folks are right. Instead, I added a message asking for help categorizing the issue